.github/workflows/stale.yaml

```diff
@@ -0,0 +1,18 @@
+name: "Maintenance: Close Stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  pull-requests: "write"
+
+jobs:
+  stale:
+    runs-on: "ubuntu-latest"
+    if: github.repository_owner == 'SchemaStore'
+    steps:
+      - uses: "actions/stale@v9"
+        with:
+          close-pr-message: "This PR was closed because it has been stalled for 365 days with no activity. Feel free make a new PR if you wish to continue¶"
```

Comment:
```suggestion
          close-pr-message: "This PR was closed because it has been stalled for 365 days with no activity. Feel free to make a new PR if you wish to continue"
```